Commands/Lists/GetListItem.cs

```diff
@@ -39,6 +39,10 @@ namespace SharePointPnP.PowerShell.Commands.Lists
         Code = "PS:> Get-PnPListItem -List Tasks -PageSize 1000",
         Remarks = "Retrieves all list items from the Tasks list in pages of 1000 items. This parameter is ignored if the Query parameter is specified.",
         SortOrder = 6)]
+    [CmdletExample(
+        Code = "PS:> Get-PnPListItem -List Tasks -PageSize 1000 -ScriptBlock { $args.Context.ExecuteQuery() } | % { $_.BreakRoleInheritance($true, $true) }",
+        Remarks = "etrieves all list items from the Tasks list in pages of 1000 items and breaks permission inheritanse on each item.",
```

Comment:
2 typos: 'etrieves' = 'Retrieves' and 'inheritanse' = 'inheritance'. If you fix these I'll happily merge this :-) It's just a matter of updating and committing your code in your own branch, and sync it to your repo, the PR will auto update.